src/main/java/com/atlauncher/gui/tabs/CreatePackTab.java

```diff
@@ -542,7 +542,7 @@ private void setupBottomPanel() {
 
     @Override
     public String getTitle() {
-        return GetText.tr("Vanilla Packs");
+        return GetText.tr("Create Pack");
```

Comment:
Not sure on this naming, but better than Vanilla Packs.

I think ideally this is just a new instance button or menu item in instances, but I think the Instances tab needs to move to a grid layout or something else for that. But that's a much bigger piece for #718